cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/partitioned/PartitionedFileSetDataset.java

```diff
@@ -614,7 +614,7 @@ public String getInputFormatClassName() {
   protected Collection<String> computeFilterInputPaths() {
     PartitionFilter filter;
     try {
-      filter = PartitionedFileSetArguments.getInputPartitionFilter(runtimeArguments, partitioning);
+      filter = PartitionedFileSetArguments.getInputPartitionFilter(runtimeArguments);
```

Comment:
previously, the filter could only contain fields that are present in the partitioning. I believe that changed now. What will happen if the filter contains an unknown field?